prime-router/src/main/kotlin/azure/ApiKeysFunctions.kt

```diff
@@ -21,6 +21,28 @@ class ApiKeysFunctions(private val settingsFacade: SettingsFacade = SettingsFaca
 
     data class ApiKeysResponse(val orgName: String, val keys: List<JwkSet>)
 
+    private fun getApiKeysForOrg(
```

Comment:
Love this change. Can you add kdoc and maybe a comment or two inside the method itself?

Reply:
Added!